### DISCUSSION TITLE
Sound works very well

### DISCUSSION THREAD

**Discussion by sonht-0555** [thenick775/gbajs3#183] [General]
@thenick775 I updated your latest core, the sound has actually improved very well.  No longer jerks when moving quickly on the screen.  How did you do that?   .... ദ്ദി ˉ͈̀꒳ˉ͈́ )⊹₊⋆ ᡣ𐭩 ....
Don't edit the sound anymore, I find it really good in `IOS 18.0` and `saveMode`

**Comment by sonht-0555** [comment]
Thank you for contributing a quality emulator to the community ^^

**Reply by thenick775** [reply]
I wouldn't be able to do it without feedback from users like you 😎 

Thanks for all the very helpful feedback along the way!

**Comment by thenick775** [comment]
I haven't changed much there for a month or two, I believe [this](https://github.com/thenick775/mgba/pull/6/files) was the last thing I changed!

Looks like all I did was up the audio buffer size, along with all the other [changes](https://github.com/thenick775/mgba/pull/5/files) that were made near that time 😎

**Comment by sonht-0555** [comment]
Exactly, increasing the buffer memory size makes the sound respond better, do you have any plans for the NBA core? @thenick775

**Reply by thenick775** [reply]
See #156, and feel free to submit a pr!
